### DISCUSSION TITLE
How do I give permission to Google to give Personal Results? Is there any way? Thank you!

### DISCUSSION THREAD

**Discussion by qwervee** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#582] [Q&A]
![image](https://user-images.githubusercontent.com/92190734/141732337-316b9c17-1459-46a7-a3b5-d9bca6797ae5.png)

**Comment by im-coder-lg** [comment]
https://gist.github.com/im-coder-lg/23fc93cbf983b36b21641563ed4f3d4e

This is the easiest method.
